src/reanimated2/animation/delay.ts

```diff
@@ -31,6 +32,15 @@ export const withDelay = function <T extends AnimationObject>(
   reduceMotion?: ReduceMotion
 ): Animation<DelayAnimation> {
   'worklet';
+
+  const a =
```

Comment:
Same comment as in `repeat.ts`, we don't need to have this condition twice

Reply:
Same as above